wafer/talks/templates/wafer.talks/talk.html

```diff
@@ -36,6 +36,15 @@ <h1>
         <a href="{% url 'wafer_user_profile' username=author.username %}">{{ author.userprofile.display_name }}</a>
       {% endfor %}
     </p>
+    {% if user.is_staff or perms.talks.view_all_talks %}
+      {% for author in object.authors.all %}
+      <p class='bio'>
+      {% blocktrans %}Bio{% endblocktrans %}{% if object.authors.count > 1 %} - {{author.userprofile.display_name}}{% endif %}:
+      {{author.userprofile.bio|default:"<em>Not informed</em>"}}
```

Comment:
I wouldn't use "Not informed". Maybe "Not provided"?

Reply:
done